qmail-remote.c

```diff
@@ -361,7 +468,11 @@ int main(int argc, char **argv)
 
  
   random = now() + (getpid() << 16);
+#ifdef SMTPUTF8
+  switch (relayhost ? dns_ip(&ip,&host) : dns_mxip(&ip,enable_utf8 ? &idnhost : &host,random)) {
+#else
   switch (relayhost ? dns_ip(&ip,&host) : dns_mxip(&ip,&host,random)) {
+#endif
```

Comment:
Line 472 is enough all by itself :-)

Reply:
And I would put it on a line distinct from the switch to make the code less convoluted.